### ISSUE TITLE
fix win32 Makefile

### ISSUE THREAD

**Issue by Mariosmsk** [OpenWaterAnalytics/EPANET#266]
I have crashed when call ENopen > ENsaveinpfile > ENclose and again ENopen.

**Comment by eladsal** [comment]
Running from VB with no issues:
```
i = ENopen(App.Path & "\net1.inp", App.Path & "\f1.rep", App.Path & "\f1.out")
i = ENsaveinpfile(App.Path & "\266.inp")
i = ENclose()

i = ENopen(App.Path & "\net1.inp", App.Path & "\f1.rep", App.Path & "\f1.out")
i = ENclose()
```

**Comment by Mariosmsk** [comment]
Closes #268
